docs/docs/winning-over-content-creators.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Winning Over Content Creators"
+issue: https://github.com/gatsbyjs/gatsby/issues/18281
+---
+
+Content creators are probably the heaviest end-users of your Gatsby projects because they create and manage the content used in your project. If you're a developer, you may find that simply talking to content creators about why _you_ like Gatsby is enough to get them interested. Below are topics a content creator cares about and how Gatsby can improve these.
+
+Some specific things that content creators care about include:
+
+- **Using tools they are familiar with.** The content creator probably has tools and workflows that they are already using, like a CMS. Gatsby could be used to integrate and possibly improve that workflow.
+- **Consistency** Consistency is key for a content creator, so their content matches the related content and branding.
+- **Audience**. Probably the biggest thing the content creator cares about is the audience, for who are they creating content?
+- **Flexible content**. Content creators care about personalizing content based on the audience that is viewing it, and creating multiple kinds of content.
+
+## Benefits
+
+Content creators may be curious about topics such as current workflow, new possibilities, flexibility.
+
+### Use the tools you already know and love
+
+> Gatsby can be configured to work with any CMS, database, or any other tools, including tools you're already using. If you love using your current CMS, great! Gatsby doesn’t decide your workflow, you do.
```

Comment:
I don't think that "any CMS or database" is actually true, so we should keep it a bit more honest. I would also remove the blockquotes since those are typically reserved for notes/callouts, and not regular paragraphs.

```suggestion
Gatsby can be configured to work with many CMS platforms and databases out of the box, with the ability to create additional source plugins for any other tools: including tools you're already using. If you love using your current CMS, great! Gatsby doesn’t decide your workflow, you do.
```